tests/test_collection.py

```diff
@@ -313,10 +314,10 @@ def test_ignore_geometry(self):
             assert("geometry" not in collection.schema.keys())
 
             feature = next(iter(collection))
-            assert(feature["properties"]["AREA"] is not None)
-            assert(feature["properties"]["STATE"] is not None)
-            assert(feature["properties"]["NAME"] is not None)
-            assert("geometry" not in feature.keys())
+            assert(feature.properties["AREA"] is not None)
+            assert(feature.properties["STATE"] is not None)
+            assert(feature.properties["NAME"] is not None)
+            assert(feature.geometry is None)
```

Comment:
Why does this need to change?

Reply:
I'd like to start paving the way for preferring attribute access to item access in the future.

Reply:
That's OK (and sounds good to me). I'm just checking this isn't a breaking change now.